base/shell/explorer/settings.cpp

```diff
@@ -73,6 +74,11 @@ BOOL TaskbarSettings::Load()
         if (sr.Position > ABE_BOTTOM)
             sr.Position = ABE_BOTTOM;
     }
+    
+    /* Policies Settings */
+    cbSize = sizeof(NULL); // dwValue was NULL at the start, is it right?
+    dwRet = SHGetValueW(hkExplorerPolicies, NULL, L"NoSimpleStartMenu", NULL, &dwValue, &cbSize);
```

Comment:
Values in the `Policies` key should be looked up using the `SHRestricted` function, not manually. You want `REST_NOSTARTPANEL` (which Wine's SHRestricted doesn't implement, but it should be straightforward to just add it to this table: https://github.com/reactos/reactos/blob/master/dll/win32/shell32/wine/shpolicy.c#L135)

Reply:
Thank you for answer, I will update code later)